atdcpp/src/atdcpp_main.ml

```diff
@@ -0,0 +1,74 @@
+(* Main *)
+
+open Atd.Import
+open Atdcpp_env
+
+let args_spec env = Arg.align
+    [ "-package",
+      Arg.String (fun x -> env := { !env with package = x }),
+      " Package name of generated files";
+      "-o",
+      Arg.String (fun x -> env := { !env with output = open_out x }),
+      " File name for C++ output"
+    ]
```

Comment:
I recommend using cmdliner, like I did in atdpy for a better user interface. Copy-paste-edit is usually the way to go.